src/index.ts

```diff
@@ -883,6 +883,27 @@ export class Bigtable {
     }
   }
 
+  /**
+   * Terminate grpc channels and close all the clients.
+   */
+  close(): Promise<void[]> {
+    const combined = [];
```

Comment:
I think to achieve what you want here we can condense the first ten lines of this function to `const combined = Object.keys(this.api).map(clientType => this.api[clientType].close())`.

Reply:
If this isn't exactly right I can try to tweak it later or help with debugging.

Reply:
This works, thank you!